husky_components/matrix/main.js

```diff
@@ -203,9 +205,9 @@ define(function() {
             this.sandbox.dom.data($tdHead, 'section', this.options.values.vertical[rowCount]);
             this.sandbox.dom.append($tr, $tdHead);
 
-            // flag for checking if every flag is true
+            // flag for checking if every flag is false
             // insert values of matrix
-            allActive = this.prepareTableColumn(
+            allInactive = this.prepareTableColumn(
```

Comment:
Actually because of the use case I don´t think `allActive` would be the better choice because I check if there is only a single one selected. That means `!allActive` is not the same as `allInactive`.